src/playercards/cards/TheRedClock.ttslua

```diff
@@ -0,0 +1,93 @@
+require("playercards/CardsWithHelper")
+local guidReferenceApi = require("core/GUIDReferenceApi")
+local playermatApi         = require("playermat/PlayermatApi")
+local searchLib            = require("util/SearchLib")
+local tokenManagerApi = require("core/token/TokenManagerApi")
+
+
+-- intentionally global
+hasXML                     = true
+isHelperEnabled            = false
+
+function onLoad(savedData)
+  -- get level via metadata and proceed accordingly:
+  -- lvl 2: one action
+  -- lvl 5: two actions
+
+  notes = JSON.decode(self.getGMNotes())
+  if savedData and savedData ~= "" then
+    local loadedData = JSON.decode(savedData)
+    isHelperEnabled = loadedData.isHelperEnabled
+  end
+  syncDisplayWithOptionPanel()
+end
+
+function updateSave()
+  self.script_state = JSON.encode({
+    isHelperEnabled = isHelperEnabled
+  })
+end
+
+function addCharge(player)
+  tokenManagerApi.addUseToCard(self, "resource")
+  Wait.frames(function() local foundTokens = searchSelfForTokens()
```

Comment:
Formatting:
Start a new line after the `function()` keyword if you can't fit the whole thing in one line (like here)